src/libpcp_web/src/webgroup.c

```diff
@@ -1867,6 +1868,39 @@ webgroup_scrape(pmWebGroupSettings *settings, context_t *cp,
     return sts < 0 ? sts : 0;
 }
 
+typedef struct webscrape {
+    pmWebGroupSettings	*settings;
+    struct context	*context;
+    sds			*msg;
+    int			status;
+    unsigned int	numnames;	/* current count of metric names */
+    sds			*names;		/* metric names for batched up scrape */
+    struct metric	**mplist;
+    pmID		*pmidlist;
+    int			numfilters;	/* current count of filters */
+    sds			*filters;
+    enum matches	match;		/* type of matching to use for filters */
+    regex_t		*regex;		/* for filter compiled regular expressions */
+    void		*arg;
+} webscrape_t;
+
+static int
+filtered(struct webscrape *filter_info, const char *name)
+{
+    int i;
+    for (i = 0; i<filter_info->numfilters; ++i) {
```

Comment:
Code style in this file IIRC is to have whitespace between 'i' and '<' and 'filter_info' here, and we tend to be the "i++" rather than "++i" style of for loop (no difference functionally, but that's the general coding style).